templates/data/opensearch/support.html

```diff
@@ -194,6 +194,49 @@ <h3 class="p-heading--5">Fast response time</h3>
     </div>
   </section>
 
+  <section class="p-section">
+    <div class="row p-section--shallow">
+      <hr class="p-rule" />
+      <div class="col-6">
+        <h2>Multiple packaging formats</h2>
+      </div>
+      <div class="col-6">
+        <p>
+          We provide maintained and supported OpenSearch packages in both snap and container formats:
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <hr class="p-rule--muted" />
+        <div class="row p-section--shallow">
```

Comment:
Same here, these should be decoupled

Reply:
Sure thing, I meant that grid classes like `row` and components like `p-section--shallow` shouldn't be used on the same element (see discussion [here](https://chat.canonical.com/canonical/pl/yjjhnfdumtf5zm9ften6pghxeo)). It should be fixed on the lines you linked as well, good catch!